README.md

```diff
@@ -97,106 +97,104 @@ def test_stays_there():
-    ```python
-    layout="""
-    ########
-    #   #  #
-    #   .  #
-    ########
-    """
-    l = create_layout(layout, bots=[(1,1), (1,2)], enemy=[(5,2), (6,2)])
-    print(l)
-    ########
-    #   #  #
-    #   .  #
-    ########
-     
-    ########
-    #0  #  #
-    #1   EE#
-    ########
-    ```
-- **`print(game)`**<br>
-    If you notice a certain configuration in a game that you want to replicate in a test, you can print the game in your move function and then use the output string as a layout in a test. For example, you could have the following move function:
-    ```python
-    def move(turn, game):
-        bot = game.team[turn]
-        # print initial state
-        if turn == 0 and bot.round == 0:
-            print(game)
-        ...
-        return (0, 0)
-    ```
-    Running this bot will print the following string on standard output:
-    ```
-    ################################
-    # .  #  . .. ..         .      #
-    #.   . .###### ##.. ##### ## # #
-    #      .    .# ## . #  #   #   #
-    # # ##..   . #.      .   . # # #
-    #.#  #### ##     .########   # #
-    #      .  .       .    # ..  # #
-    ## ######. ## ###         .#   #
-    #   #.         ### ## .###### ##
-    # #  .. #    .       .  .      #
-    # #   ########.     ## ####  #.#
-    # # # .   .      .# .   ..## # #
-    #   #   #  # . ## #.    .      #
-    # # ## ##### ..## ######. .   .#
-    #      .         .. .. .  #  . #
-    ################################
-    
-    ################################
-    #    #                        1#
-    #       ###### ##   ##### ## #0#
-    #            # ##   #  #   #   #
-    # # ##       #             # # #
-    # #  #### ##      ########   # #
-    #                      #     # #
-    ## ######  ## ###          #   #
-    #   #          ### ##  ###### ##
-    #E#     #                      #
-    # #E  ########      ## ####  # #
-    # # #             #       ## # #
-    #   #   #  #   ## #            #
-    # # ## #####   ## ######       #
-    #                         #    #
-    ################################
-    ```
-    Now you can copy and paste this string in a test, pass it to `setup_test_game` and verify that your bot returns the move you were expecting.
+Given a layout string, returns a [Game](#the-game-object) instance suitable to be passed to a [move](#the-move-function) function. Printing a `Game` object will print its layout string. In the simplest form a layout string is a multiline string where the character `#` identifies walls, `.` the food pellets, `E` the enemy bots (you must have two of them for a layout to be legal), and `0` and `1` representing the bots in your team corresponding to turn `0` and `1`.
+
+For example a maze `8x4` with our bots in `(1, 1)` and `(1, 2)`, where the enemies are on `(5,2)` and `(6,2)` and food pellets in `(2, 2)` and `(6,1)`, and an additional wall in `(4,1)` will look like this:
```

Comment:
What happened to the indentation?

Reply:
i removed it since this is not a list of functions any more. was that not correct?

Reply:
I don’t know. Just wanted to make sure it was not an oversight.